stages/org.osbuild.dracut

```diff
@@ -195,6 +195,8 @@ def main(tree, options):
 
         subprocess.run(["/usr/sbin/chroot", tree,
                         "/usr/bin/dracut",
+                        "--no-hostonly",
+                        "--reproducible",
```

Comment:
We already have an option for that (line 147), I think it should just switch to `True` for a default.

Reply:
Oh, I missed that. Thanks for spotting it, I changed it.